notebooks/Tracking_drift_check.ipynb

```diff
@@ -0,0 +1,885 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    df.columns
</pre><p>I guess that these cells are more exploratory, right? With that said, I would remove them to make the notebook shorter/cleaner to someone to use. If they want to check the values, they can create the cells themselves.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>